stack.yaml

```diff
@@ -1,16 +1,16 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/d673b92f2a2d9354d102514d0a0fa74f8248b14a/snapshot.yaml
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/faf5ba333c7e6f1710a355d5af0abbaec696dfd1/snapshot.yaml
```

Comment:
Can we not change the resolver/snapshot too often? Because changing it means that all packages need to be recompiled, which is not a good use of our time.

Reply:
How often is not "too often"?

Reply:
Ideally only when we really need a new version of a package that is only included in a new snapshot :slightly_smiling_face:

Reply:
Have you never been in a situation where you have debugged a problem only to find it was already fixed in a later version?

Reply:
Yes.

But was that the reason for updating the snapshot now? :slightly_smiling_face:

Reply:
so that if we hit any problems, it is unlikely to be a problem that is already fixed in a later version of the LTS.